.storybook/test-runner.ts

```diff
@@ -6,7 +6,7 @@ const config: TestRunnerConfig = {
   postRender: async page => {
     // We wait for the page to load for at least one second
     // as there's no reliable way globally to ensure everything loaded correctly
-    await page.waitForTimeout(1000);
+    await page.waitForTimeout(500);
```

Comment:
Why do you decrease time ??

Reply:
Why are you making a big deal out of this? 😓 this is called experimentation.

Reply:
I just wanted to know. Why reduce the time, because if it's too short, it could block testing, which is a long process.

Reply:
What you're saying makes no sense at all. The reason we have a longer timeout is because some of our Components right now are doing asynchronous work that shouldn't be done the way it is done.

E.g.: the Author and Codebox component. Those will be changed in the future.

Reply:
Ok ! Thanks for explaining. It’s make sense.